pkg/features/kube_features.go

```diff
@@ -739,6 +739,14 @@ const (
 	// Enables kubelet support to size memory backed volumes
 	SizeMemoryBackedVolumes featuregate.Feature = "SizeMemoryBackedVolumes"
 
+	// owner: @alexanderConstantinescu
+	// kep: http://kep.k8s.io/3458
+	// beta: v1.27
+	//
+	// Enables less load balancer re-configurations by the service controller
+	// (KCCM) as an effect of changing node state.
+	StableLoadBalancerNodeSet featuregate.Feature = "StableLoadBalancerNodeSet"
```

Comment:
do we need this here?
It doesn't seem we need it here since the cloud-provider has its own features gates and are the ones that we are importing

Reply:
I don't know. I thought this was the centralized location for all feature gates in kube and staging/src/k8s.io/controller-manager/pkg/features/kube_features.go the feature gates for the controller-manager, which can't import and reference `pkg/featuress/kube_features.go`

Reply:
I admit I don't know the protocol here.  pkg/features/kube_features.go obviously governs things in the k/k Go module, but since staging repos are "independent" (just pretend :), they can't rely on k/k/pkg/....